K2Bridge/Visitors/RangeClauseVisitor.cs

```diff
@@ -53,7 +53,7 @@ public void Visit(RangeClause rangeClause)
             }
         }
 
-        private static void FillKqlQuery(RangeClause rangeClause, Func<string, string> valueConverter = null)
+        private void FillKqlQuery(RangeClause rangeClause, Func<string, string> valueConverter = null)
```

Comment:
why static was removed?

Reply:
Because `schemaRetriever` is a member in the class, so `EncodeKustoField` isn't static, so this can't be